src/Assembly.js

```diff
@@ -116,10 +115,26 @@ class Records {
 @observer
 class Assembly extends React.Component {
   @observable currentPage = Home
-  @observable coordinator = null
+  @observable coordinator = {
+    patients: [
+      {
+        med_report_status: 'notreported',
+        id: "pete",
+        took_medication: 'Yes',
+        firstname: "Peter",
+        lastname: "Campbell",
+        phone: 15304120086,
+        treatment_start_date: null,
+        last_repored_date: null,
+        side_effects: ["Nausea", "Redness"],
+        percent_since_start: 48,
+        photo: [],
+        patient_note: [],
+        coordinator_note: [],
+      },
+    ]
+  }
 
-  // TODO change from `User` to `Account`
-  // Records("User", "user").first
   @observable account = {
```

Comment:
I've seen user, account, and participant. Are they different or can we standardize?